runtime/runtime/src/actions.rs

```diff
@@ -154,6 +154,9 @@ pub(crate) fn action_function_call(
         Some(VMError::InconsistentStateError(err)) => {
             return Err(StorageError::StorageInconsistentState(err.to_string()).into());
         }
+        Some(VMError::CacheError(err)) => {
+            return Err(StorageError::StorageInconsistentState(err).into());
```

Comment:
We probably need another error in case we run it against a challenge. But it's okay for now